### DISCUSSION TITLE
How to create a sub-theme that inherits from sphinx-book-theme?

### DISCUSSION THREAD

**Discussion by leifwalsh** [executablebooks/meta#1126] [Q&A] [answered]
https://sphinx-book-theme.readthedocs.io/en/stable/contributing/subthemes.html#defining-your-own-css suggests it is possible (but maybe challenging) to create a sub-theme. In my case, I'd just like to add a little CSS (changing colors) that I can share among several projects.

I think I've got things mostly scaffolded right, but one thing I'm stuck on is that https://sphinx-book-theme.readthedocs.io/en/stable/contributing/subthemes.html#defining-your-own-css says I should be able to `@import "sphinx-book-theme.css"` from my `index.scss`, but when I try to compile my theme, it says it can't find that.

I'm also not clear on how to "inherit" sphinx-book-theme's `index.js` from my own. Here, in theory I think I want to import `sphinx-book-theme.js`, but can't figure out how to get it on node's module path.

I know this isn't "officially supported", but the docs suggest it's possible. Does anyone have any tips I can try, or know of an existing theme that does this correctly?

**Comment by choldgraf** [comment, marked as answer]
Here's an example of a theme that inherits from the book theme. Does this help you get started?

https://github.com/2i2c-org/sphinx-2i2c-theme

**Reply by leifwalsh** [reply]
That's **perfect**, thank you so much!
